src/System.IO.FileSystem.AccessControl/tests/TempDirectory.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Runtime.InteropServices;
```

Comment:
is there a reason you couldn't use `src\Common\tests\System\IO\TempFile.cs` and `src\Common\tests\System\IO\TempDirectory.cs` ?

you just include them in your project
```
        <Compile Include="$(CommonTestPath)\System\IO\TempFile.cs">
          <Link>Common\System\IO\TempFile.cs</Link>
        </Compile>
```